index.html

```diff
@@ -221,5 +221,33 @@ <h3>
         </div>
       </div>
     </section>
+
+    <section class="contact-form">
+      <h2>Contact me</h2>
+      <p>
+        If you have an application you are interested in developing, a feature
+        that you need built or a project that needs coding. I’d love to help
+        with it
+      </p>
+
+      <form action="https://formspree.io/f/xyyognlv" method="POST">
+        <input
+          type="text"
+          name="name"
+          maxlength="30"
+          placeholder="Your Name"
+          required
+        />
+        <input type="email" name="email" placeholder="Email Address" required />
+        <textarea
+          name="message"
+          placeholder="write your message here"
```

Comment:
- [ ] [OPTIONAL] As you started your other placeholder with capital letter, why don't you use it also here `Write your message here`. Dont forget to do  to it next time. ok? 😃🙌